src/routes/protected.tsx

```diff
@@ -485,13 +486,27 @@ export const protectedRoutes: RouteNode[] = [
           },
           {
             path: EnrollmentDashboardRoutes.CURRENT_LIVING_SITUATIONS,
-            // No perm needed because it only requires enrollment visibility
-            element: <EnrollmentCurrentLivingSituationsPage />,
+            element: (
+              <EnrollmentRoute
+                permissions={[]} // Only requires enrollment visibility
```

Comment:
Could pass `canViewEnrollments`?

Reply:
We can't pass an empty array, since it makes the `hasPermissions` check evaluate as false. I made `permissions` optional on the `EnrollmentRoute` and `ProjectRoute` instead.

Reply:
Ok, sounds good